pipenv/utils.py

```diff
@@ -535,7 +535,9 @@ def resolve_deps(deps, which, which_pip, project, sources=None, verbose=False, p
     markers_lookup = {}
 
     python_path = which('python', allow_global=allow_global)
-    backup_python_path = shellquote(sys.executable)
+    backup_python_path = sys.executable
+    if os.name == 'nt':
```

Comment:
My only caveat is that we don't currently use shellquoting for `sys.executable` when we invoke it in other places, even for windows, so I believe you can skip this alternative logic -- I think you are probably assuming it was added for a reason, but in this case I think it was just added as a mistake :)

Reply:
Cool, I'll drop the if.